dinero/gateways/authorizenet_gateway.py

```diff
@@ -553,7 +553,9 @@ def create_customer(self, options):
             if error_code == 'E00039':  # Duplicate Record
                 e.customer_id = None
 
-                customer_match = re.search(r'^A duplicate record with ID (.*) already exists.$', e.message[0][1])
+                customer_match = re.search(
+                    r'A duplicate record with ID (.*) already exists.', str(e)
+                )
```

Comment:
Well, the issue is that in python3 `e.message` doesn't exist, so the options are either to access `e.args` or `str(e)`. I couldn't see a way to use `e.args` that would work in both python2 and python3, so the options are either to break it into cases, or just look at the `str`. I'm not super happy with this, but it seems like the simpler of the two options?